proxy/http/Http1ServerSession.h

```diff
@@ -77,15 +74,64 @@ class Http1ServerSession : public VConnection
   void do_io_shutdown(ShutdownHowTo_t howto) override;
 
   void reenable(VIO *vio) override;
+  void
+  new_connection(NetVConnection *new_vc, MIOBuffer *iobuf, IOBufferReader *reader) override
+  {
+    // TODO: refactor this
+    ink_assert(0);
+  }
+  void
+  start() override
+  {
+    // TODO: refactor this
+    ink_assert(0);
+  }
+  void
+  attach_server_session(Http1ServerSession *ssession, bool transaction_done = true) override
+  {
+    // TODO: refactor this
+    ink_assert(0);
+  }
+  void
+  release(ProxyTransaction *trans) override
+  {
+    // TODO: refactor this
+    ink_assert(0);
+  }
```

Comment:
Is this a functional intermediate step?  I assume the original versions of these methods are still around and are being used instead of the new stubs?

Reply:
These stubs are the undefined virtual functions of ProxySession. Some existing functions matched the name but not the signature. 
Yes, this an intermediate step. Note the "// TODO: refactor this"